activefires_pp/post_processing.py

```diff
@@ -329,7 +329,9 @@ def get_global_mask_from_shapefile(shapefile, lonlats, start_geom_index=0):
     shape = geometry.geoms[0]
     pth = Path(shape.exterior.coords)
     mask = pth.contains_points(points)
-    for shape in geometry.geoms[start_geom_index:]:
+
+    constituent_part = geometry.geoms[start_geom_index:]
+    for shape in constituent_part.geoms:
         pth = Path(shape.exterior.coords)
         mask = np.logical_or(mask, pth.contains_points(points))
 
```

Comment:
this looks very similar to `get_maks_from_mulitpolygon`, can it be factorized?

Reply:
Yes, true. I think this should be done okay now, and at least one test is covering this